org-code-javabuilder/lib/src/main/java/org/code/javabuilder/CodeBuilder.java

```diff
@@ -1,37 +1,27 @@
 package org.code.javabuilder;
 
-import java.io.*;
+import java.io.File;
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
-import org.code.protocol.*;
-import org.code.protocol.LoggerUtils.ClearStatus;
-import org.code.protocol.LoggerUtils.SessionTime;
+import org.code.protocol.GlobalProtocol;
+import org.code.protocol.InternalErrorKey;
+import org.code.protocol.JavabuilderException;
+import org.code.protocol.OutputAdapter;
 
 /** The orchestrator for code compilation and execution. */
 public class CodeBuilder implements AutoCloseable {
```

Comment:
Do we still need to implement AutoCloseable if we aren't using the `close()` method?

Reply:
Yep we still do since we use it in a try-with-resources block.

Reply:
Does it continue to need to be in a try-with-resources block?

Reply:
Hmm, yeah I guess not. I'll move it out.